src/compiler/program.ts

```diff
@@ -1420,8 +1420,9 @@ export function getImpliedNodeFormatForFileWorker(
     packageJsonInfoCache: PackageJsonInfoCache | undefined,
     host: ModuleResolutionHost,
     options: CompilerOptions,

```

Comment:
shouldnt options passed in here be options for file? = `redirectOptions ?? options` instead of having to pass the redirect options.

Reply:
See https://github.com/microsoft/TypeScript/pull/60039/files/5a97e07ce383c44c7a73e76a1143f8499a507c7e#diff-353e1c0017b5e0e0791b61eed3d20b219f5690f81aeb41fc333d76424c723175